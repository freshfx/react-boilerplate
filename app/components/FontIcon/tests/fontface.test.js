import loadFonts from '../fontFace'

describe('fontFace', () => {
  it('should match the snapshot', () => {
    loadFonts()
    const [head] = document.getElementsByTagName('head')
    const [style] = head.children
    expect(style.innerHTML).toMatchSnapshot()
  })
})
