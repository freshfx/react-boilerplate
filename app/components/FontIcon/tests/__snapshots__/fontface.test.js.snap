// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FontIcon fontFace should match the snapshot 1`] = `
"
  @font-face {
    font-family: 'icons';
    src:  url('IMAGE_MOCK');
    src:  url('IMAGE_MOCK#iefix') format('embedded-opentype'),
      url('IMAGE_MOCK') format('truetype'),
      url('IMAGE_MOCK') format('woff'),
      url('IMAGE_MOCK#icons') format('svg');
    font-weight: normal;
    font-style: normal;
  }
"
`;
